### DISCUSSION TITLE
Провайдер билайн блочит запрет

### DISCUSSION THREAD

**Discussion by theseus2908** [Flowseal/zapret-discord-youtube#950] [Q&A]
Провайдер билайн блочит запрет. Пофиксите пожалуйста

**Comment by screwsen** [comment]
У меня провайдер не билайн, но со второй половины дня перестал работать обход блокировки ютуба. Иконки видео загружаются, а само видео уходит в бесконечную загрузку. Обход дискорда работает, скачан последний релиз, в файлах ничего не менял, так как итак все работало. Помогите пж, что делать?

**Comment by n0niKo** [comment]
у меня тоже провайдер билайн но у меня сразу закрывается батник но на не сколько секунд открывается вот этот фаил что делать 
![изображение_2024-12-17_162638865](https://github.com/user-attachments/assets/f6705c49-8bc5-4e2e-a1a4-89502731ce28)

**Comment by Jeaden123** [comment]
а у меня та же хня на ростелекоме, только грузит через альт 4 или МГТС2, если говорим об одновременной загрузке дс и ютуба, а через все остальные идеальнее всего лишь ютуб грузит. Походу автору пора выкатывать апдейт), ибо п*доры в ркн спустя три месяца нашли лазейки

**Reply by beyn999** [reply]
Это правда! У меня все работала, но со вчера не работал генерал через ростелком. Щас затестил через 4 альт и работает действительно)

**Reply by Jeaden123** [reply]
> Это правда! У меня все работала, но со вчера не работал генерал через ростелком. Щас затестил через 4 альт и работает действительно)

ну, будем надеяться что Флоусиал уже в курсе событий)

**Reply by Jeaden123** [reply]
> Это правда! У меня все работала, но со вчера не работал генерал через ростелком. Щас затестил через 4 альт и работает действительно)

вообще я пока ещё обновил у себя резеврный способ на прокси через амнезию, ибо через альт4 и мгтс2 хоть и грузит дс отлично, ютуб через них грузит херово, какой то видос нормально идёт а какой то грузится миллион лет пхаха. Иногда просто психую и смотрю их через прокси и не парюсь)
